README.md

```diff
@@ -238,13 +238,15 @@ based on [awesome-python](https://github.com/vinta/awesome-python/).
 * [nefertari](https://github.com/ramses-tech/nefertari) -  Nefertari is a REST
   API framework sitting on top of Pyramid and ElasticSearch.
 * [pyramid_swagger](https://github.com/striglia/pyramid_swagger) - Convenient
-  tools for using Swagger to define and validate your interfaces in a Pyramid webapp.
+  tools for using Swagger to define and validate your interfaces in a Pyramid webapp. (SWagger 2.0 document)
+* [pyramid-openapi3](https://github.com/niteoweb/pyramid_openapi3) - Validate Pyramid views against an OpenAPI 3.0 document. Similar to pyramid_swagger but for openapi3
```

Comment:
FWIW, we are in the process of migrating this repo under the Pylons Project.

Also "openapi3" should be "OpenAPI 3.0".

Add a period to the end of the sentence.

Reply:
Done.